doc/contributing/security-steward-on-off-boarding.md

```diff
@@ -6,6 +6,7 @@
   to the project and not to use/disclose to their employer.
 * Add them to the security-stewards team in the GitHub nodejs-private
   organization.
+* Add them to the [public website team](https://github.com/orgs/nodejs/teams/website).
```

Comment:
Does there need to be a corresponding off boarding step?

Reply:
I'm on the fence. It might be hard to determine whether someone is should be left on the website group for other reasons. It may be better to leave it to a regular website member pruning process (automated or human-powered) which is something we don't currently have and need to figure out, actually.